node-process-argv/process.js

```diff
@@ -0,0 +1 @@
+console.log({ process });
```

Comment:
I'm curious why you put `{}` around `process` before you logged it.

Reply:
It said print the entire process object so I assumed it wanted an object passed in it. It is fixed now!

Reply:
The {} is not needed to create an object, since `process` is already an object. Look over the difference in the log output between the two and see if you can figure out why they differ.